lib/wrapbox/runner/ecs.rb

```diff
@@ -245,7 +245,13 @@ def create_task(task_definition_arn, class_name, method_name, args, command, par
         begin
           run_task_options = build_run_task_options(task_definition_arn, class_name, method_name, args, command, cl, launch_type, parameter.environments, parameter.task_role_arn)
           @logger.debug("Task Options: #{run_task_options}")
-          resp = client.run_task(run_task_options)
+
+          begin
+            resp = client.run_task(run_task_options)
+          rescue Aws::ECS::Errors::ThrottlingException
```

Comment:
[imo] Adding jitter to retry intervals is better to avoid throttling exceptions. All threads retry to create their task at the same time with the current implementation.
cf. [Exponential Backoff And Jitter | AWS Architecture Blog](https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/)

Reply:
I've implemented. 02d125e